demo/src/app.js

```diff
@@ -360,7 +361,12 @@ const AppContent = ({ language, onLanguageClick }) => {
             key3: 'row2_val3',
             notClickable: true,
         },
-        { key1: 'row3_val1', key2: 'row3_val2', key3: 'row3_val3' },
+        {
+            key1: 'row3_val1',
+            key2: 'row3_val2',
+            key3: 'row3_val3',
+            notClickable: true,
```

Comment:
not in the scope of this PR

Reply:
yes you need it to test combinations on hover and rowdark rowlight